app/components/summary_list_component.html.erb

```diff
@@ -1,9 +1,11 @@
 <dl class="govuk-summary-list">
   <% rows.each do |row| %>
     <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= row[:key] %>
-      </dt>
+      <% if row[:key] %>
```

Comment:
We should refactor this away, because a definitions list without a `dt` is weird.